src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -571,6 +570,10 @@ private async Task ValidateHostRuntime(bool isInProc, bool isVerbose)
             throw new CliException($"Invalid host runtime '{HostRuntime}'. Valid values are 'default', '{DotnetConstants.InProc8HostRuntime}', '{DotnetConstants.InProc6HostRuntime}'.");
         }
 
+        private void PrintMessageForInProcSdk()
+        {
+            Console.WriteLine($"This version of the Azure Functions Core Tools requires your project to reference version {DotnetConstants.InProcFunctionsMinSdkVersion} or later of {DotnetConstants.InProcFunctionsSdk}. Please update to the latest version. For more information, see: {DotnetConstants.InProcFunctionsDocsLink}");
```

Comment:
Will this print to the desired output stream? In earlier iterations, we had it as an exception that should go to stderr. This would just show up in stdout, right? I think this should be error level, as it is a terminal state.

Reply:
In my discussion with Fabio, we talked about not throwing an exception here, instead just log and exit. But we can discuss if you prefer an error here.  @fabiocav any thoughts/preference on this?

Reply:
Updating to throw cli exception here